docs/40_examples.md

```diff
@@ -69,3 +69,48 @@ You can customize the way your Project is deployed:
 ```sh
 miactl deploy development --no-semver --revision tags/v1.0.0
 ```
+
+## Deploy Project using a service account from a CD pipeline
```

Comment:
I'm not sold to import this example inside the repository of miactl. The link to the public docs will be broken here because it link to a non existent page.

I would prefer a more extensive example to setup a context with a service account.

Reply:
Up

Reply:
in regards of the doc link, if you use the complete url the doc aggregator will resolve it when opening the PR, so you can safely use: https://docs.mia-platform.eu/docs/development_suite/identity-and-access-management/manage-service-accounts

Reply:
I finally added a more exhaustive explanation and I also fixed the link. Let me know if you want other additions.